src/MeetingsSDKAdapter.js

```diff
@@ -818,6 +827,69 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     return concat(getDisplayData$, sharingEvents$);
   }
 
+  /**
+   * Attempts to toggle roster to the given meeting ID.
+   * A roster toggle event is dispatched.
+   *
+   * @private
+   * @param {string} ID ID of the meeting to toggle roster
+   */
+  async handleRoster(ID) {
+    const sdkMeeting = await this.fetchMeeting(ID);
```

Comment:
Does this return a promise? You're [using the same here](https://github.com/webex/sdk-component-adapter/pull/99/files#diff-67d6f2edd2655b86027c2d6d69ae37ba5e655b9f49c43d042f86a2c8c2b981faR858) without the async. I think [by the definition it just returns an object](https://github.com/webex/sdk-component-adapter/blob/7a9f87826d1b303856ecbe8678ff064f946e435d/src/MeetingsSDKAdapter.js#L374)